packages/cid-verifier/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "cid-verifier",
+  "version": "1.0.0",
+  "description": "Verify that a CID's content is safe for the web",
+  "private": true,
+  "type": "module",
+  "main": "./dist/worker.mjs",
+  "scripts": {
+    "lint": "standard",
+    "build": "tsc && node scripts/cli.js build",
+    "predev": "node scripts/cli.js ipfs --start",
+    "dev": "miniflare dist/worker.mjs --watch --debug -m",
+    "postdev": "node scripts/cli.js ipfs --stop",
+    "test": "npm run test:setup && npm-run-all -p -r mock:google-cloud.io test:worker",
+    "test:worker": "ava --verbose test/*.spec.js",
+    "test:setup": "npm run build",
+    "mock:google-cloud.io": "smoke -p 9111 test/mocks/google-cloud.io --set foo"
```

```suggestion
    "mock:google-cloud.io": "smoke -p 9111 test/mocks/google-cloud.io"
```

we don't need this right?